src/content/en/tools/chrome-devtools/profile/network-performance/understanding-resource-timing.markdown

```diff
@@ -0,0 +1,222 @@
+---
+layout: shared/narrow
+title: "Understanding Resource Timing"
+description: "It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues."
+published_on: 2016-02-04
+order: 2
+authors:
+  - jonathangarbee
+translation_priority: 0
+key-takeaways:
+  timing:
+    - "Understand the phases of resource timing."
+    - "Know what each phase provides to the Resource Timing API."
+    - "Realize different indicators of performance problems in the timeline graph. Such as series of transparent bars or large green chunks."
+notes:
+  note-tbd:
+    - "TBD note."
+---
+
+<p class="intro">
+  It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues.
+</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.emulate-network %}
+
+All network requests are considered resources.
+As they are retrieved over the network, resources have distinct lifecycles expressed in terms of resource timing.
+The network panel uses the same [Resource Timing API](http://www.w3.org/TR/resource-timing) that is available to application developers.
+
+The Resource Timing API provides a rich level of detail about each individual assets time to be received.
+The primary phases of the request lifecycle are:
+
+* Redirect
+  * Immediately begins `startTime`.
+  * If a redirect is happening, `redirectStart` begins as well.
+  * If a redirect is occurring at the end of this phase then `redirectEnd` will be taken.
+* App Cache
+  * If it’s application cache fulfilling the request, a `fetchStart` time will be taken.
+* DNS
+  * `domainLookupStart` time is taken at the beginning of the DNS request.
+  * `domainLookupEnd` time is taken at the end of the DNS request.
+* TCP
+  * `connectStart` is taken when initially connecting to the server.
+  * If TLS or SSL are in use then `secureConnectionStart` will start when the handshake begins for securing the connection.
+  * `connectEnd` is taken when the connection to the server is complete.
+* Request
+  * `requestStart` is taken once the request for a resource has been sent to the server.
+* Response
+  * `responseStart` is the time when the server initially responds to the request.
+  * `responseEnd` is the time when the request ends and the data is retrieved.
+
+![Resource Timing API diagram](imgs/resource-timing-api.png)
+
+## Viewing in DevTools
+
+To view the full timing information for a given entry of the network panel you have three options.
+
+1. You may hover the timing graph under the timeline column. This will present a popup that shows the full timing data.
```

Comment:
Drop the "You may" here and in the items below. `Hover over the timing graph under the timeline column.`